packages/config/src/forkConfig/index.ts

```diff
@@ -139,14 +145,23 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
       return sszTypesFor(forkName);
     },
     getMaxBlobsPerBlock(fork: ForkName): number {
-      switch (fork) {
-        case ForkName.fulu:
-          return config.MAX_BLOBS_PER_BLOCK_FULU;
-        case ForkName.electra:
-          return config.MAX_BLOBS_PER_BLOCK_ELECTRA;
-        default:
-          return config.MAX_BLOBS_PER_BLOCK;
+      if (!isForkPostDeneb(fork)) {
+        throw Error(`getMaxBlobsPerBlock is not avaiable for fork ${fork}`);
+      }
+      const forkEpoch = this.getForkInfoFromFork(fork).epoch;
+      const blobSchedule = config.BLOB_SCHEDULE;
+
+      if (blobSchedule.length === 0) {
+        throw Error("Attempt to get MAX_BLOBS_PER_BLOCK from empty BLOB_SCHEDULE");
+      }
+
+      for (const entry of config.BLOB_SCHEDULE) {
+        if (forkEpoch === entry.EPOCH) {
+          return entry.MAX_BLOBS_PER_BLOCK;
+        }
       }
+
+      return config.BLOB_SCHEDULE[0].MAX_BLOBS_PER_BLOCK;
```

Comment:
why do we need all of this?

Reply:
What are you referring to? I have updated the code to match https://github.com/ethereum/consensus-specs/blob/7c5b431f40b71bedc2cc5a07a16d8be45d8abfe1/specs/fulu/das-core.md#get_max_blobs_per_block

Reply:
Updated